docs/includes/apiargs-MongoDBCollection-common-option.yaml

```diff
@@ -0,0 +1,79 @@
+arg_name: option
+name: bypassDocumentValidation
+type: boolean
+description: |
+   If ``true``, allows the write operation to circumvent document level
+   validation. Defaults to ``false``.
+interface: phpmethod
+operation: ~
```

Comment:
@schmalliso noted that `operation`, while a required key in the Python model, is only used for replacing `{{role}}` in content. I decided to use `operation: ~` (i.e. null) for common option/param definitions and will probably do so for all definitions.